PHP_CodeSniffer/Tokenizers/GraphQL.php

```diff
@@ -0,0 +1,258 @@
+<?php
+/**
+ * Copyright © Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace PHP_CodeSniffer\Tokenizers;
+
+use GraphQL\Language\Lexer;
+use GraphQL\Language\Source;
+use GraphQL\Language\Token;
+
+/**
+ * Implements a tokenizer for GraphQL files.
+ */
+class GraphQL extends Tokenizer
+{
+
+    /**
+     * Defines how GraphQL token types are mapped to PHP token types.
+     *
+     * @var array
+     */
+    private $tokenTypeMap = [
+        Token::AT           => 'T_DOC_COMMENT_TAG',
+        Token::BANG         => null, //TODO Should we map this to a specific type
```

Comment:
Need to finalize the mapping.

Reply:
Fixed in https://github.com/magento/magento-coding-standard/pull/141/commits/43ef0a4b8d14197f8ec34cc97161a45a6d71b45a